pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -556,13 +555,20 @@ func (jm *ControllerV2) syncCronJob(
 	case errors.HasStatusCause(err, corev1.NamespaceTerminatingCause):
 	case errors.IsAlreadyExists(err):
 		klog.InfoS("Job already exists", "cronjob", klog.KRef(cj.GetNamespace(), cj.GetName()), "job", klog.KRef(jobReq.GetNamespace(), jobReq.GetName()))
+		var getError error
+		jobResp, getError = jm.jobControl.GetJob(jobReq.Namespace, jobReq.Name)
+		if getError != nil {
+			return cj, nil, getError
+		}
```

Comment:
Don't do that, if a job was created by a different actor (other worker) then it'll also have taken care of updating status and we shouldn't do anything with it. Just continue without taking any actions.

Reply:
+1, removed